<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWGS</class>
 <widget class="QDialog" name="DialogWGS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Longitude [deg]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLon</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxLat">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>125</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxLon">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Latitude [deg]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLat</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Altitude [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxAlt</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxAlt">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBoxRev">
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxLat</tabstop>
  <tabstop>spinBoxLon</tabstop>
  <tabstop>spinBoxAlt</tabstop>
  <tabstop>checkBoxRev</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogWGS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogWGS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
